app/models/annotation/AnnotationUploadService.scala

```diff
@@ -12,7 +12,7 @@ import javax.inject.Inject
 import models.annotation.nml.NmlResults._
 import models.annotation.nml.{NmlParser, NmlResults}
 import net.liftweb.common.{Box, Empty, Failure, Full}
-import net.liftweb.util.Helpers.tryo
+import net.liftweb.common.Box.tryo
```

Comment:
This is an import redirect inside of liftweb-util. By always including from liftweb-common, I was able to completely remove the libtweb-util dependency.